pkg/runtime/runtime_test.go

```diff
@@ -6422,7 +6422,7 @@ func TestStartReadingFromBindings(t *testing.T) {
 		err := rt.startReadingFromBindings()
 
 		assert.NoError(t, err)
-		assert.Len(t, mockAppChannel.Calls, 1)
+		assert.True(t, mockAppChannel.AssertCalled(t, "InvokeMethod", mock.Anything, mock.Anything))
```

Comment:
I think `AssertCalled` already causes the test to fail if false. Don't think the `assert.True` is needed